02-peer-protocol.md

```diff
@@ -215,6 +215,10 @@ The receiving node MUST NOT:
 
 #### Rationale
 
+The requirement for `funding_satoshi` to be less than 2^24 satoshi is a self-imposed limit while implementations are not yet considered stable, and temporary.
```

Comment:
Nit: I suggest changing from "… is a self-imposed limit while implementations are not yet considered stable, and temporary" to "… is a temporary self-imposed limit while implementations are not yet considered stable".

Reply:
> is a temporary recommendation for implementations until the network is considered stable.